examples/echo_server/migrations/5_run_log.js

```diff
@@ -1,13 +1,20 @@
-let chainlinkDeployer = require("../chainlink_deployer.js");
-let LinkToken = artifacts.require("../node_modules/smartcontractkit/chainlink/solidity/contracts/LinkToken.sol");
-let Oracle = artifacts.require("../node_modules/smartcontractkit/chainlink/solidity/contracts/Oracle.sol");
-let RunLog = artifacts.require("./RunLog.sol");
+const clmigration = require("../clmigration.js");
+const request = require("request-promise");
+const LinkToken = artifacts.require("../node_modules/smartcontractkit/chainlink/solidity/contracts/LinkToken.sol");
+const Oracle = artifacts.require("../node_modules/smartcontractkit/chainlink/solidity/contracts/Oracle.sol");
+const RunLog = artifacts.require("./RunLog.sol");
 
-module.exports = function(truffleDeployer) {
-  truffleDeployer.then(async () => {
-    await chainlinkDeployer.job("only_jobid_logs_job.json").then(async function(body) {
-      console.log(`Deploying Consumer Contract with JobID ${body.id}`);
-      await truffleDeployer.deploy(RunLog, LinkToken.address, Oracle.address, body.id);
-    }).catch(console.log);
-  });
+let url = "http://chainlink:twochains@localhost:6688/v2/specs";
+let job = {
+  "_comment": "A runlog has a jobid baked into the contract so chainlink knows which job to run.",
+  "initiators": [{ "type": "runlog" }],
+  "tasks": [
+    { "type": "HttpPost", "url": "http://localhost:6690" }
+  ]
 };
+
+module.exports = clmigration(async function(truffleDeployer) {
```

Comment:
Please note that only migrations wrapped with `clmigration` are skipped in the test environment. I didn't wrap all migrations, just ones reaching out to external APIs and any migrations running afterwards. This ended up being 4 total across all examples and solidity.